docs/development/update-strategy-periodic.md

```diff
@@ -0,0 +1,42 @@
+# Periodic update strategy 
+
+The agent supports a `periodic` strategy, which allows gating reboots based on "reboot windows", defined on weekly basis.
+
+This strategy is a port of [locksmith reboot windows][locksmith], with a few differences:
+
+ * all times and dates are UTC-based to avoid skews and ambiguities
```

Comment:
nit: It may be worth noting it also uses the 24 hour clock format (IE: 01:00 == 1 am UTC). Most people will infer this with UTC, but still.

Reply:
Ack, good thing to note. It was true for locksmith as well and it's a different thing than UTC timezone, so I've put that info on a dedicated sentence on its own.